Library/Formula/sdl_ttf.rb

```diff
@@ -18,6 +18,10 @@ class SdlTtf < Formula
   depends_on "sdl"
   depends_on "freetype"
 
+  # Fix broken TTF_RenderGlyph_Shaded()
+  # https://bugzilla.libsdl.org/show_bug.cgi?id=1433
```

Comment:
`TTF_RenderGlyph_Shaded()` has been refactored and reimplemented in `sdl2_ttf` so this patch is no longer relevant there. It seems `sdl2_ttf` has diverged quite significantly since the last release of `sdl_ttf` despite 0.0.1 difference in version.